authority/authority.go

```diff
@@ -91,8 +91,11 @@ type Authority struct {
 
 	adminMutex sync.RWMutex
 
-	// Do Not initialize the authority
+	// If true, do not initialize the authority
 	skipInit bool
+
+	// If true, does not output initialization logs
```

Comment:
```suggestion
	// If true, do not output initialization logs
```
To match the comments above.